core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -233,6 +228,11 @@ package object config {
   private[spark] val EXECUTOR_HEARTBEAT_MAX_FAILURES =
     ConfigBuilder("spark.executor.heartbeat.maxFailures").internal().intConf.createWithDefault(60)
 
+  private[spark] val EXECUTOR_PROCESS_TREE_METRICS_ENABLED =
+    ConfigBuilder("spark.executor.processTreeMetrics.enabled")
+      .booleanConf
+      .createWithDefault(false)
```

Comment:
shouldn't we add .doc for this metric and also spark.eventLog.logStageExecutorMetrics.enabled in addition to what is added to configuration.md?

Reply:
Added.